include/libopencm3/stm32/f7/dsi.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2016 Chuck McManis <cmcmanis@mcmanis.com>
```

Comment:
as elsewhere, I don't feel this is a correct copyright statement for a single include in a new file.